fastats/maths/correlation/pearson.py

```diff
@@ -1,6 +1,8 @@
 
 import numpy as np
 
+from fastats.scaling.scaling import standard
+
 
 def pearson(x, y):
```

Comment:
I wonder if these non-pairwise corrs can be expressed by pairwise? All we'd have to do is to construct a 2-D array ad-hoc

Reply:
That's possible, but there's a performance cost